### ISSUE TITLE
Change Table of contents heading to h2, include French translation or title prop, change to sentence case

### ISSUE THREAD

**Issue by krischarbonneau** [DTS-STN/Service-Canada-Design-System#688]
Based on the design pattern shown [here](https://design.canada.ca/common-design-patterns/tables.html), this should be an `<h2>` tag instead of a `<p>` tag. The current implementation affects a11y since navigating through the headings would see a screen reader user skipping over the table of contents.

https://github.com/DTS-STN/Service-Canada-Design-System/blob/973bac037913346c22637ed66ccb2b8e912743d1/src/components/TableContent/TableContent.js#L11

**Comment by krischarbonneau** [comment]
To add to the above, the text should follow sentence case and remove the colon, as follows:

`On this page`

Currently, the text is also hardcoded so when switching to French on other applications, English text is still shown. This should be updated to be a prop, or added to translations and the component adjusted accordingly.

**Comment by krischarbonneau** [comment]
Fixed in #704
